app/routes/bank/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { PageHeader } from "~/components/elements/PageHeader"
+import { redirectIfUserNotLoggedIn } from "~/services/auth/session.server"
+
+export const loader = async (request: Request) => {
+  await redirectIfUserNotLoggedIn(request)
```

Comment:
for some reason, when I navigate directly to http://localhost:3000/bank, this line throws an error. Specifically, the error message is

```
TypeError: Cannot read properties of undefined (reading 'get')
    at getCookieSession (C:\Users\user\Documents\Work\dchp3\app\services\auth\session.server.ts:28:71)...
```

Reply:
Thanks for the catch. Dumb error.